gateway-ha/src/main/java/io/trino/gateway/ha/config/ClusterStatsMonitorType.java

```diff
@@ -18,5 +18,6 @@ public enum ClusterStatsMonitorType
     NOOP,
     INFO_API,
     UI_API,
-    JDBC
+    JDBC,
+    JMX
```

Comment:
Could you explain why existing monitor types don't work well in your environment?

Reply:
Thanks for your explanation. 
Note that we shouldn't document it if it's the temporary solution. Please revret changes in `docs/installation.md`.

Reply:
Done